app/src/components/utility/RawData/RawData.tsx

```diff
@@ -27,6 +28,20 @@ const parseJSON = (JSONString: string) => {
 export const RawData: React.FC<RawDataProps> = ({ rawData }) => {
   const { t } = useTranslation();
   const rawDataJSON: object = parseJSON(rawData);
+  const [stringLength, setStringLength] = useState<number>();
+
+  const windowWidth = window.innerWidth;
+
+  useEffect(() => {
+    if (windowWidth <= MOBILE_BREAKPOINT) {
+      const MobileRawDataStringLength = Math.floor(windowWidth * 0.035);
```

Comment:
Where are the coefficients `0.035` and `0.0595` coming from?

Reply:
They are working values spanning my largest display to smallest screen.  I was hoping each of us could take a look at the raw data display on own devices to make the values a little more precise.  I left a bit of wiggle room either way.

Reply:
So you played around with a few numbers and found that these worked?

Is it not possible to solve this with CSS instead of JS?

https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
For context: the eventual strategy was to set `word-break: break-all;` on the `.string-value` class

Reply:
@Thistle-Strum there should be other places in the app that use this method. I would try global searching for `word-break` or `break-all` and you should be able to find some examples.